### PR TITLE
gltf_viewer: add stats and view controls.

### PR DESCRIPTION
**Pull request by prideout**

### CODE REVIEW

File: libs/gltfio/include/gltfio/SimpleViewer.h [hunk 1]
```diff
@@ -102,6 +102,11 @@ class SimpleViewer {
      */
     int getSidebarWidth() const { return mSidebarWidth; }
 
+    /**
+     * Allows clients to inject custom UI.
+     */
+    void setCustomUI(std::function<void()> callback) { mCustomUI = callback; }
```
Comment (romainguy) [inline libs/gltfio/include/gltfio/SimpleViewer.h]: `setUiCallback` maybe?

File: libs/gltfio/include/gltfio/SimpleViewer.h [hunk 2]
```diff
@@ -125,6 +131,8 @@ class SimpleViewer {
     bool mShowWireframe = false;
     bool mEnableSunlight = true;
     bool mEnableDithering = true;
+    bool mEnablePrepass = true;
+    bool mEnableFxaa = true;
```
Comment (romainguy) [inline libs/gltfio/include/gltfio/SimpleViewer.h]: You should also add a toggle for MSAA 4x

File: samples/app/FilamentApp.h [hunk 3]
```diff
@@ -80,6 +80,8 @@ class FilamentApp {
 
     void setSidebarWidth(int width) { mSidebarWidth = width; }
 
+    size_t getSkippedFrames() const { return mSkippedFrames; }
```
Comment (romainguy) [inline samples/app/FilamentApp.h]: `getSkippedFramesCount()`?

File: libs/gltfio/include/gltfio/SimpleViewer.h (hunk 1 above)
Comment (prideout) [reply libs/gltfio/include/gltfio/SimpleViewer.h]: Done

File: libs/gltfio/include/gltfio/SimpleViewer.h (hunk 2 above)
Comment (prideout) [reply libs/gltfio/include/gltfio/SimpleViewer.h]: Done

File: samples/app/FilamentApp.h (hunk 3 above)
Comment (prideout) [reply samples/app/FilamentApp.h]: Done

**Pull request merged**